drivers/destination_report/app/models/destination_report/details.rb

```diff
@@ -147,6 +148,7 @@ def client_headers
     def client_columns
       [
         c_t[:id],
+        c_t[:PersonalID],
```

Comment:
One possible complicating factor.  Is this a `source_client` or `destination_client`  If these are destination clients, we'll need to fetch the PersonalIDs of their source clients...  The goal would be to get the PersonalID related to the exit, so, maybe better `ex_t[:PersonalID]` (that'd be fine as long as we have an exit record somewhere.

Reply:
The Exit table is within the scope where these are being referenced and the ex_t arel is loading data in both the html and the excel export. I'm updating this to use the exit arel instead of the client arel.